README.md

```diff
@@ -973,6 +998,40 @@ and run:
 $ cargo run --release --bin server
 ```
 
+At first, we are going to test our server using [Insomnia][insomnia].
+Insomnia is a free API testing tool, and it supports testing gRPC services.
+After you downloaded and installed Insomnia, open it and click ⨁ icon and select «gRPC Request».
+
+![grpc_request.png](images%2Fgrpc_request.png)
+
+Enter `localhost:9001` in gRPC server bar. 
+In gRPC method dropdown, select _Click to use server reflection_.
+
+![server_reflection.png](images%2Fserver_reflection.png)
+
+Select Unary method `/Inventory/Add` and add body of json request. 
+
+```json
+{
+   "identifier": {
+      "sku": "TESTSKU"
+   },
+   "stock": {
+      "price": 1.9900000095367432,
+      "quantity": 20
+   },
+   "information": {
+      "name": "bananas",
+      "description": "yellow fruit"
+   }
+}
+```
+
+![inso.gif](images%2Finso.gif)
+
+> **Note**: Feel free to experiment with other methods of Inventory service on your own.
+
+Now let's test gRPC client using cli app we developed earlier.
```

Comment:
It seems like this whole section should be _before_ we start building the client, after we build the server no?

like a step specifically just to make sure the server is working, then we can go on and add the CLI?

Because at this point, we already have a tool that can do everything we're demonstrating here so it almost feels a bit redundant, but as a tool to initially test that our service is building and running correctly it makes sense, wdyt?